MapboxAndroidDemo/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.application'
 apply from: "$project.rootDir/gradle/script-git-version.gradle"
 apply plugin: 'com.google.firebase.firebase-perf'
+apply plugin: 'com.github.triplet.play'
```

Comment:
if release integration will not happen as part of this pr, please remove any parts of it